infrasim/model/elements/pci_imc.py

```diff
@@ -37,13 +43,17 @@ def init(self):
         self.id = self.__imc_info["id"]
         self.bus = self.__imc_info["bus"]
         self.addr = self.__imc_info.get("addr")
+        self.imc_slot_topo = self.__imc_info.get("imc_slot_topo")
```

Comment:
missing declaration in __init__() function. 
this is unsafe.

Reply:
code updated, please help to check the latest changes, thanks